File: VB-Examples/20_Security/DetermineDocumentEncryption.vb
Imports System.IO
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents
Imports Spire.Doc.Fields

Namespace DetermineDocumentEncryption
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim isEncrypted As Boolean = Document.IsEncrypted("..\..\..\..\..\..\Data\TemplateWithPassword.docx")
			If isEncrypted = True Then
				MessageBox.Show("This document is encrypted. ")
			Else
				MessageBox.Show("This document is unencrypted. ")
			End If
		End Sub

	End Class
End Namespace

File: CS-Examples/20_Security/DetermineDocumentEncryption.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;
using Spire.Doc.Fields;

namespace DetermineDocumentEncryption
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            bool isEncrypted = Document.IsEncrypted(@"..\..\..\..\..\..\Data\TemplateWithPassword.docx");
            if(isEncrypted == true)
            {
                MessageBox.Show("This document is encrypted. ");
            }
            else
            {
                MessageBox.Show("This document is unencrypted. ");
            }
        }

    }
}
